### PR TITLE
on backend registered, initialize terminals

### LINKED ISSUE #159639: Task reconnection status does not work for default build task
Recording 2022-08-31 at 08 20 47

I urge to selfhost on this for a bit. Status bar never shows the state for me, I think never has after window reload.

### PR DESCRIPTION
**Pull request by meganrogge**
fixes https://github.com/microsoft/vscode/issues/159639

### CODE REVIEW

File: src/vs/workbench/contrib/terminal/browser/terminalService.ts [hunk 1]
```diff
@@ -280,9 +280,7 @@ export class TerminalService implements ITerminalService {
 				this._connectionState = TerminalConnectionState.Connected;
 			}
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/browser/terminalService.ts:282]: this was only used by the `terminalView` before, so is safe to remove in favor of calling `_initializeTerminals` within the `terminalService` itself

**Comment by meganrogge** [comment]
working on the cause of the failure

**Comment by meganrogge** [comment]
there's still a timing issue where an extra terminal gets created... investigating

**Comment by meganrogge** [comment]
Failing tests though doesn't seem to have an issue when I test manually 🤔

**Comment by Tyriar** [comment]
> Failing tests though doesn't seem to have an issue when I test manually

@meganrogge the tests are failing only in browser and remote, you're probably running local tests

**Comment by meganrogge** [comment]
new approach is needed as this does not work

**Comment by meganrogge** [comment]
I realized that likely the issue here is `renderBody` isn't happening unless the panel is visible @Tyriar do you know how to change this?

**Pull request closed without merging**